packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/LocalCompare.js

```diff
@@ -8,73 +8,89 @@ import BaseCompare from './BaseCompare';
 const log = logger('wdio-visual-regression-service:LocalCompare');
 
 export default class LocalCompare extends BaseCompare {
-  constructor(options = {}) {
+  constructor(options) {
     super(options);
-    this.getScreenshotPath = options.screenshotName;
-    this.getReferencefile = options.referenceName;
-    this.getDiffFile = options.diffName;
-    this.misMatchTolerance = _.get(options, 'misMatchTolerance', 0.01);
-    this.ignoreComparison = _.get(options, 'ignoreComparison', 'nothing');
+
+    this.ignoreComparison = 'ignore';
+    this.misMatchTolerance = 0.01;
   }
 
+  /**
+   * Process for Local Comparison of a new latest screenshot against a reference screenshot, if one exists. If the
+   * two images are of a different size or are not within the  mismatch tolerance, a screenshot highlighting the
+   * differences will be created.
+   *
+   * @param {Object} context - Information provided to process the screenshot.
+   * @param {Object} context.browser - Contains the browser's name, version, userAgent.
+   * @param {Object} context.suite - The test suite that is running.
+   * @param {Object} context.test - The test that is running.
+   * @param {Object} context.meta - Contains the element, exclude, hide, remove, viewport as meta data to use.
+   * @param {*} base64Screenshot - The screenshot captured by the selenium command to process.
+   */
   async processScreenshot(context, base64Screenshot) {
-    const screenshotPath = this.getScreenshotPath(context);
-    const referencePath = this.getReferencefile(context);
+    const {
+      referencePath,
+      latestPath,
+      diffPath,
+    } = this.getScreenshotPaths(context);
 
-    await fs.outputFile(screenshotPath, base64Screenshot, 'base64');
+    // create latest screenshot
+    await fs.outputFile(latestPath, base64Screenshot, 'base64');
 
-    const referenceExists = await fs.exists(referencePath);
+    const referenceExists = fs.existsSync(referencePath);
 
     if (referenceExists) {
       log.info('reference exists, compare it with the taken now');
```

Comment:
Would this get printed to the terminal every time? If so would that cause too much noise? 

```suggestion
      log.info('reference screenshot exists, compare it with the taken screenshot now');
```